system_tests/pubsub.py

```diff
@@ -169,7 +169,12 @@ def _by_timestamp(message):
         self.assertEqual(message2.data, MESSAGE_2)
         self.assertEqual(message2.attributes['extra'], EXTRA_2)
 
+    def _maybe_emulator_skip(self):
+        if os.getenv(PUBSUB_EMULATOR) is not None:
+            self.skipTest('IAM not supported by Pub/Sub emulator')
+
```

Comment:
Wouldn't this be clearer / more idiomatic using `unittest2.skipIf('PUBSUB_EMULATOR'in os.environ)`?